components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -379,6 +379,8 @@ void RewardsServiceImpl::OnLoad(SessionID tab_id, const GURL& url) {
   if (baseDomain == "")
     return;
 
+  const std::string publisher_url = origin.scheme() + "://" + baseDomain + "/";
```

Comment:
an actual fix

Reply:
note that this would return `http:///` / `https:///` for all IP address URLs